decidim-design/app/views/layouts/decidim/design/_head.html.erb

```diff
@@ -1,5 +1,8 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <%= javascript_pack_tag "decidim_core", "decidim_design", defer: false %>
-<%= stylesheet_pack_tag "decidim_core", "decidim_design", media: "all" %>
+
+<%= append_stylesheet_pack_tag "decidim_overrides", media: "all" %>
```

Comment:
Shouldn't we do an specific override file for this module too (`decidim_design_overrides`)?

Reply:
Ok, this is not a hill worth dying on IMO 😄